src/main/java/dukeApp/parse/Parse.java

```diff
@@ -0,0 +1,55 @@
+package dukeApp.parse;
+import dukeApp.files.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Parse {
+    String statement;
+
+    public Parse(String statement) {
+        this.statement = statement;
+    }
+
+    public boolean decode(TaskList tasks) {
```

Comment:
The method return type is misleading, as I would have thought that valid commands return true, since the method name is decode. Instead, only bye returns true